utils/docker/Dockerfile.ubuntu.20.04

```diff
@@ -17,6 +17,7 @@ ARG UID=1000
 # It's better to put the apt-get update in the same "cache layer" as the
 # apt-get install command so that the apt database is updated if/when the
 # installed packages list below is updated
+RUN ls
```

Comment:
What is this doing?

Reply:
Oops.   I added it when I was trying to debug the Ubuntu build locally as a sanity check.  I'll remove it in follow on patch